charts/judge-api/values.yaml

```diff
@@ -0,0 +1,77 @@
+replicaCount: 1
+
+image:
+  repository: us-east4-docker.pkg.dev/judge-395516/judge-image-registry/judge-api
+  pullPolicy: IfNotPresent
+  tag: "v1.1.0"
+  pullSecrets:
+    - name: gcr-secret
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: false
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
```

Comment:
We should template good default securityContext values across all deployments. Example [here](https://github.com/goharbor/harbor-helm/blob/main/values.yaml#L472).

Reply:
we should sometime